run-e2e.sh

```diff
@@ -13,18 +13,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# run the script to run e2e tests for cluster loader, netperf and dns. 
+# run the script to run e2e tests for cluster loader, netperf and dns.
 # This enables the same to be invoked from kubetest from test-infra.
 
 set -o nounset
 set -o pipefail
 
 PERFTEST_ROOT=$(dirname "${BASH_SOURCE}")
-#CLUSTERLOADER
-cd ${PERFTEST_ROOT}/clusterloader/e2e/ && go test -c -o e2e.test
-./e2e.test --ginkgo.v=true --ginkgo.focus="Cluster\sLoader" --kubeconfig="${HOME}/.kube/config" --viper-config=../config/test
-#NETPERF
-cd ${PERFTEST_ROOT}/network/benchmarks/netperf/ && go run ./launch.go  --kubeConfig="${HOME}/.kube/config" --hostnetworking --iterations 1
-#DNS
-cd ${PERFTEST_ROOT}/dns
- ./run --params params/kubedns/default.yaml --out-dir out --use-cluster-dns
+while [ ! $# -eq 0 ]
```

Comment:
Why do you need while loop?

Reply:
As an iteration ,if one test gets completed it will prompt for next iteration tests.
If you prefer , I am totally agreed to remove the loop.

Reply:
I suggest removing it to keep it simple (also because it maybe used by automated CI tests).

Also I suggest using double brackets (`[[ ]]`) instead of single ones (as that's more safer in bash iirc)